<ui version="4.0" >
 <class>ChangePasswordDialogClass</class>
 <widget class="QDialog" name="ChangePasswordDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>228</height>
   </rect>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="styledFrame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <property name="spacing" >
       <number>5</number>
      </property>
      <property name="leftMargin" >
       <number>1</number>
      </property>
      <property name="topMargin" >
       <number>5</number>
      </property>
      <property name="rightMargin" >
       <number>1</number>
      </property>
      <property name="bottomMargin" >
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <property name="topMargin" >
         <number>5</number>
        </property>
        <property name="rightMargin" >
         <number>4</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>10</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="closeToolButton" >
          <property name="focusPolicy" >
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip" >
           <string>CloseHint</string>
          </property>
          <property name="accessibleName" >
           <string>transparentButton</string>
          </property>
          <property name="layoutDirection" >
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="popupMode" >
           <enum>QToolButton::DelayedPopup</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel" >
        <property name="text" >
         <string>Title</string>
        </property>
        <property name="indent" >
         <number>9</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="mainDialogFrame" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth" >
         <number>0</number>
        </property>
        <layout class="QGridLayout" name="gridLayout" >
         <property name="leftMargin" >
          <number>50</number>
         </property>
         <property name="topMargin" >
          <number>15</number>
         </property>
         <property name="rightMargin" >
          <number>50</number>
         </property>
         <property name="bottomMargin" >
          <number>15</number>
         </property>
         <property name="horizontalSpacing" >
          <number>11</number>
         </property>
         <property name="verticalSpacing" >
          <number>9</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="currentPasswordLabel" >
           <property name="text" >
            <string>CurrentPassword</string>
           </property>
           <property name="buddy" >
            <cstring>currentPasswordEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="currentPasswordEdit" >
           <property name="echoMode" >
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <spacer name="verticalSpacer_4" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>150</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="newPasswordLabel" >
           <property name="text" >
            <string>NewPassword</string>
           </property>
           <property name="buddy" >
            <cstring>newPasswordEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="newPasswordEdit" >
           <property name="echoMode" >
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="repeatNewPasswordLabel" >
           <property name="text" >
            <string>NewPassword2</string>
           </property>
           <property name="buddy" >
            <cstring>repeatNewPasswordEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QLineEdit" name="repeatNewPasswordEdit" >
           <property name="echoMode" >
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <property name="topMargin" >
         <number>9</number>
        </property>
        <property name="rightMargin" >
         <number>9</number>
        </property>
        <property name="bottomMargin" >
         <number>9</number>
        </property>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="okButton" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="accessibleName" >
           <string>rectButton</string>
          </property>
          <property name="text" >
           <string>OkButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="accessibleName" >
           <string>rectButton</string>
          </property>
          <property name="text" >
           <string>CancelButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0" />
 <resources/>
 <connections/>
</ui>
